### PR TITLE
Feature/21891 migrate to java 11

### PR DESCRIPTION
**Pull request by dtsarenko**

### CODE REVIEW

File: src/test/java/com/icthh/xm/tmf/ms/customer/repository/timezone/DateTimeWrapper.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
+package com.icthh.xm.tmf.ms.customer.repository.timezone;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "jhi_date_time_wrapper")
+public class DateTimeWrapper implements Serializable {
```
Comment (ishkurko) [inline src/test/java/com/icthh/xm/tmf/ms/customer/repository/timezone/DateTimeWrapper.java]: Why we need this wrapper?

File: src/main/jib/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (ishkurko) [inline src/main/jib/entrypoint.sh]: Plz, use unified `entrypoint.sh` with K8s support and docker secrets.

File: src/test/java/com/icthh/xm/tmf/ms/customer/repository/timezone/DateTimeWrapper.java (hunk 1 above)
Comment (dtsarenko) [reply src/test/java/com/icthh/xm/tmf/ms/customer/repository/timezone/DateTimeWrapper.java]: I think, this entity is added to jhipster tests to check that hibernate saves time in utc format.

Comment (dtsarenko) [reply src/test/java/com/icthh/xm/tmf/ms/customer/repository/timezone/DateTimeWrapper.java]: https://github.com/jhipster/generator-jhipster/pull/8501

File: src/main/jib/entrypoint.sh (hunk 2 above)
Comment (dtsarenko) [reply src/main/jib/entrypoint.sh]: this jib/entrypoint.sh is generated by jhipster for the gradle plugin com.google.cloud.tools.jib for local development. For travis buid I added /docker/Dockerfile and /docker/ entrypoint.sh

Comment (dtsarenko) [reply src/main/jib/entrypoint.sh]: @ishkurko

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,6 @@
+#Thu Nov 14 17:00:44 EET 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```
Comment (vkirichenko) [inline gradle/wrapper/gradle-wrapper.properties]: I suggest to use the latest version of gradle 
https\://services.gradle.org/distributions/gradle-6.0.1-bin.zip
also this version is generated by latest version of jhipster generator

File: src/main/java/com/icthh/xm/tmf/ms/customer/web/rest/CustomerDelegate.java [hunk 4]
```diff
@@ -21,6 +22,7 @@
     @LogicExtensionPoint(value = "RetrieveCustomer", resolver = ProfileKeyResolver.class)
     @PreAuthorize("hasPermission({'id': #id}, 'CUSTOMER.GET')")
     @Override
+    @PrivilegeDescription("Privilege to get a customer")
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/tmf/ms/customer/web/rest/CustomerDelegate.java]: Privilege to get a customer list

File: build.gradle [hunk 5]
```diff
@@ -4,113 +4,115 @@ buildscript {
...
+    id 'war'
     //jhipster-needle-gradle-plugins - JHipster will add additional gradle plugins here
 }
 
-spotbugs {
-    toolVersion = '3.1.3'
-    ignoreFailures = true
-}
+group = "com.icthh.xm.tmf.ms.customer"
+version = "2.0.0"
```
Comment (sergeysenja1992) [inline build.gradle]: move to property file
version=2.0.0

**Pull request merged**